packages/ui/src/components/va-toast/VaToast.demo.vue

```diff
@@ -34,22 +34,63 @@
       <button @click="handleToast">Open Toast</button>
       <button @click="$vaToast.close(handleId)">Close {{handleId}}</button>
     </VbCard>
+    <VbCard title="useToast hook">
+      <button @click="hookTest">Start test</button>
+    </VbCard>
   </VbDemo>
 </template>
 
 <script>
 import { useGlobalConfig } from '../../services/global-config/global-config'
+import { useToast } from './useToast'
 
 export default {
   setup () {
     const { setGlobalConfig } = useGlobalConfig()
-
     setGlobalConfig(config => ({
       ...config,
       VaToast: {
         color: 'red',
       },
     }))
+
+    const { init, close, closeAllCreatedInThisHook, closeAll } = useToast()
```

Comment:
I think we should either change variable names or never destructure toast object. Standalone "init" function looks too vague.

Reply:
User can change it's name
```ts
const { init: initToast, close, closeAllCreatedInThisHook, closeAll } = useToast()
```
or simply

```ts
const toast = useToast()
```


I just left it as it was in options API `$vaToast.init()`